roles/calico_master/README.md

```diff
@@ -20,7 +14,22 @@ To install, set the following inventory configuration parameters:
 * `openshift_use_openshift_sdn=False`
```

Comment:
Had to set `l_osm_default_node_selector: ''` to prevent the master from injecting a role=app nodeselector into the daemonsets.

Reply:
Alternatively, one can add a empty node selector to the `kube-system` project.